kedro/extras/datasets/README.md

```diff
@@ -1,5 +1,9 @@
 # Datasets
 
+> **Warning**

```

Comment:
So if I understand correctly, the deletion of `kedro.extras.datasets` has not been done in `develop`?

Reply:
That's correct. We haven't done any of the breaking changes for `0.19.0`, because that would make `develop` and `main` diverge. Long story short this became a nightmare for the `0.18.0` release so we're taking a different approach this time. I'd be happy to explain more if needed!